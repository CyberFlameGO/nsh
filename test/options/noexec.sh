echo "you can see this message"
set -n
echo "hidden message"
