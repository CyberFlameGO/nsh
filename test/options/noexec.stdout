you can see this message
